From e86c43b86179fba90a1d9dd5acb554767af6740f Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Mon, 12 Jun 2023 15:23:55 +0100
Subject: [PATCH] dts: bcm2710-rpi-zero-2-w: Remove WLAN firmwares

With careful use of qualified firmware names there is no need for the
ability to override the device names based on Device Tree properties.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm/boot/dts/bcm2710-rpi-zero-2-w.dts | 13 -------------
 1 file changed, 13 deletions(-)

--- a/arch/arm/boot/dts/bcm2710-rpi-zero-2-w.dts
+++ b/arch/arm/boot/dts/bcm2710-rpi-zero-2-w.dts
@@ -161,19 +161,6 @@
 	brcmf: wifi@1 {
 		reg = <1>;
 		compatible = "brcm,bcm4329-fmac";
-
-		firmwares {
-			fw_43436p {
-				chipid = <43430>;
-				revmask = <4>;
-				fw_base = "brcm/brcmfmac43436-sdio";
-			};
-			fw_43436s {
-				chipid = <43430>;
-				revmask = <2>;
-				fw_base = "brcm/brcmfmac43436s-sdio";
-			};
-		};
 	};
 };
 
